addons/attributes/XEH_postInit.sqf

```diff
@@ -26,3 +26,11 @@
         "reloadSpeed"
     ];
 }] call CBA_fnc_addEventHandler;
+
+[QGVAR(setAllHitPointsDamage), {
```

Comment:
Perhaps renamed to just `setHitPointsDamage`?

Reply:
Avoided that because of https://community.bistudio.com/wiki/setHitPointDamage